### ISSUE TITLE
LB health checks failing on v1.5.1

### ISSUE THREAD

**Issue by m1pl** [kubeflow/gcp-blueprints#376]
I deployed a kubeflow cluster following the docs with the v1.5.1 tag. Everything was successful, but instead of the dashboard I see
```
Error: Server Error
The server encountered a temporary error and could not complete your request.
Please try again in 30 seconds.
```
The LB in GCE shows this:
![Screenshot from 2022-07-22 13-41-38](https://user-images.githubusercontent.com/4653607/180432169-091b2c88-e445-4816-bcbc-2537506c440b.png)

The exact same deployment with tag v1.5.0 is working fine.

Any help would be appreciated.

**Comment by skrawn** [comment]
I ran into the same issue, you need to patch two files to fix the health check for backend service: https://github.com/kubeflow/gcp-blueprints/issues/371

**Comment by gkcalat** [comment]
This is the same as #371. It was fixed in #374.

You can pull the latest version from [master branch](https://github.com/kubeflow/gcp-blueprints/tree/master) and follow [the deployment instructions](https://www.kubeflow.org/docs/distributions/gke/deploy/) or wait for [Kubeflow v1.6 release](https://github.com/kubeflow/gcp-blueprints/issues/365), which is expected in August 2022.
